lib/samson/secrets/shared/list_recursive.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
```

Comment:
`shared` folder seems kinda overkill 🤷‍♂️

Reply:
I think it's nice to have, and makes it obvious what lives there.